docker-compose.yaml

```diff
@@ -0,0 +1,27 @@
+version: '3'
+services:
+  php:
+    build: ./docker/php
+    volumes:
+      - .:/src
+    working_dir: /src
+    environment:
+      - TEST_RABBITMQ_HOST=rabbitmq
+      - TEST_PROXY_HOST=proxy
+      - TEST_PROXY_PORT=5673
+    links:
+      - rabbitmq
+    depends_on:
+      - rabbitmq
+      - proxy
+
+  rabbitmq:
+    image: rabbitmq:3.5
```

Comment:
What version of RabbitMQ does this image install? If it's RabbitMQ 3.5.X, that is very, very old. Can the latest one be used?

Reply:
Yes that would install 3.5.x

https://hub.docker.com/_/rabbitmq

Reply:
That's too old, please use the latest available. Does Docker support something like `rabbitmq:latest` so that we don't have to keep modifying this file?

Reply:
Yep it does. That should be fine for most scenarios I reckon.

Reply:
changed to latest